projects/ngx-openlayers/src/lib/attribution.component.ts

```diff
@@ -1,18 +1,15 @@
 import { Component, ElementRef, OnInit } from '@angular/core';
-import { Attribution } from 'ol/control';
 
 @Component({
   selector: 'aol-attribution',
   template: '<ng-content></ng-content>',
 })
 export class AttributionComponent implements OnInit {
-  instance: Attribution;
-  html: string;
+  label: string;
 
   constructor(private elementRef: ElementRef) {}
 
   ngOnInit() {
-    this.html = this.elementRef.nativeElement.innerHTML;
-    this.instance = new Attribution(this);
+    this.label = this.elementRef.nativeElement.innerHTML;
```

Comment:
As far as I can tell, this component is only used in conjunction with `AttributionsComponent` to call `setAttributions()`, which takes an `AttributionLike` (which is a type alias for `string | string[] | (FrameState): (string|Array<string>)`), so using the `Attribution` control here was incorrect.

As such, I modified this and `AttributionsComponent` to work with `string`s instead.